Fix triple-quote guidance to be about style of semantically equal code (#71)
invenia/BlueStyle: README.md

    arr = [1, 2, 3,]
    ```

> - Triple-quotes and triple-backticks written over multiple lines should have closing quotes indented once.

Reviewer: ```suggestion
- Triple-quotes and triple-backticks written over multiple lines should be indented.
```

>   Since triple-quotes use the indentation of the lowest indented line excluding the opening quotes, the least indented line in the string should be aligned with the closing quotes (i.e. also indented once).

Reviewer: ```suggestion
  As triple-quotes use the indentation of the lowest indented line (excluding the opening quotes) the least indented line in the string or ending quotes should be indented once.
```

Although unusual I think we'd be ok with the second example but not the first:
```julia
julia> s = """
       hello
           """
"hello\n    "

julia> s = """
           hello
               """
"hello\n    "
```

        hello
        world!
        """
    cmd = ```
        program
            --flag value
            parameter
        ```

    # No:
    str = """
    hello
    world!
    """
    cmd = ```
          program
              --flag value
              parameter
          ```
    ```

> - Assignments using triple-quotes or triple-backticks should have the opening quotes on the same line assignment operator.

Author: ```suggestion
- Assignments using triple-quotes or triple-backticks should have the opening quotes on the same line as the assignment operator.
```
